testapps/php56_e2e/composer.json

```diff
@@ -1,6 +1,15 @@
 {
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/stanley-cheung/Protobuf-PHP"
```

Comment:
OK gotcha. I advocate making it the default, since it will be very difficult for users to discover the need for them to override with your fork on their own.